webapp/src/main/webapp/js/newsComposerPlugin.js

```diff
@@ -25,5 +25,28 @@
     }
   };
 
+  const hintActivityComposerPlugin = {
```

Comment:
Your latest commit should be reverted IMO. We are in the News project, so it is better to name the plugin with a meaningful name, not a generic one.

Reply:
ok